zulipterminal/ui_tools/buttons.py

```diff
@@ -20,10 +20,20 @@ def __init__(self, controller: Any, caption: str,
                  prefix_character: Union[str, Tuple[Any, str]]='\N{BULLET}',
                  text_color: Optional[str]=None,
                  count: int=0) -> None:
+        if isinstance(prefix_character, tuple):
+            prefix = prefix_character[1]
+        else:
+            prefix = prefix_character
```

Comment:
This may be one line, but it reaches ~98 character line length. At that point it's often clearer with a normal `if`, as much as I do like the one-liners.

Also, I do think that the repeated line but with a `[1]` on the end of one, is actually quite clear.

Reply:
Sure. I can understand the though process. Looks good :+1: